package.json

```diff
@@ -52,7 +51,8 @@
     "package-ntp-sponsored-images": "node ./scripts/ntp-sponsored-images/package",
     "package-ntp-super-referrer": "node scripts/packageNTPSuperReferrerComponent.js",
     "package-ntp-super-referrer-mapping-table": "node scripts/packageNTPSuperReferrerMappingTableComponent.js",
-    "package-youtubedown": "node ./scripts/packageYoutubedown.js",
+    "package-youtubedown": "node ./scripts/packagePlaylist.js",
+    "package-playlist": "node ./scripts/packagePlaylist.js",
```

Comment:
we can create a PR on devops to change the pipeline now to use `package-playlist` now, so that you can remove line `"package-youtubedown": "node ./scripts/packagePlaylist.js",` also. We then can merge this PR first, then apply the change to devops PR right after

Reply:
@linhkikuchi done, see https://github.com/brave/devops/pull/11196